src/Spago/CLI.hs

```diff
@@ -321,6 +333,7 @@ parser = do
 
     projectCommands = CLI.subcommandGroup "Project commands:"
       [ initProject
+      , script
```

Comment:
"project commands" are meant to be "commands to do things in a project", so it feels like this belongs more to "other commands" since you wouldn't use this in a project

Reply:
Gotcha, thanks for clearing that up, I wasn't sure. I'll update it!